library/sources/http-server/shouldDiscoverRoute.ts

```diff
@@ -1,7 +1,7 @@
-import { extname } from "path";
+import { getFileExtension } from "../../helpers/getFileExtension";
 
 const EXCLUDED_METHODS = ["OPTIONS", "HEAD"];
-const IGNORE_EXTENSIONS = ["properties", "php", "asp", "aspx", "jsp", "config"];
+const IGNORE_EXTENSIONS = ["properties", "config"];
```

Comment:
```suggestion
// These are extensions that are too long to get ignored, we manually ignore them
const IGNORE_EXTENSIONS = ["properties", "config"];
```
Or something like that